src/accounts/Deal.ts

```diff
@@ -128,6 +141,18 @@ export class Deal {
 		return DealStatus.IN_PROGRESS;
 	}
 
+	isPending() {
+		return this.status === DealStatus.PENDING;
+	}
+
+	isClosed() {
+		return this.status === DealStatus.CLOSED;
+	}
+
+	isInProgress() {
+		return this.status === DealStatus.IN_PROGRESS;
+	}
+
 	get daysRemaining() {
 		if (!this.goLiveAt || this.status === DealStatus.PENDING) {
```

Comment:
This is duplicate business logic. If we ever change how a pending deal is defined, we should only have to change it in one place.

```suggestion
		if (!this.goLiveAt || this.isPending()) {
```

Reply:
Same suggestion in regard to closed deals, but GitHub doesn't allow me to add a comment there.
change `if (this.status === DealStatus.CLOSED) {` to `if (this.isClosed()) {`